IHP/IDE/SchemaDesigner/View/Columns/Edit.hs

```diff
@@ -114,7 +114,9 @@ typeSelector :: Maybe PostgresType -> [Text] -> Html
 typeSelector postgresType enumNames = [hsx|
         <select id="typeSelector" name="columnType" class="form-control select2-simple">
             {option selected "TEXT" "Text"}
-            {option selected "INT" "Int"}
+            {option selected "SMALLINT" "Int (16bit)"}
+            {option selected "INT" "Int (32bit)"}
```

Comment:
Can we keep this one just `Int`? Otherwise new users might be confused on what Int type to pick :)

```suggestion
            {option selected "INT" "Int"}
```

Maybe we can also then move small and big ints near the end of the list (the list is sorted by general frequency of usage)?